src/Chapter1.hs

```diff
@@ -620,7 +646,11 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = theLastDigit + secondLastDigit
+  where
+    theLastDigit = abs n `mod` 10
+    secondLastDigit = (abs n `mod` 100 - theLastDigit) `div` 10
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that is actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(theLastDigit, secondLastDigit) = x `divMod` 10
```

You can see how we could pattern match on the pair 🙂